### ISSUE TITLE
Fix support for SOPS encrypted variables

### ISSUE THREAD

**Issue by tleguijt** [labd/mach-composer#191]
Since MACH 2.0 SOPS encrypted variables doesn't work correctly anymore.
To reproduce;

- create a `secrets.yml` and encrypt with SOPS
- include secrets in MACH configuration: 
```yaml
---
mach_composer:
  version: "1.2.2"
  variables_file: secrets.yml
```
- run `mach-composer plan`

Following error occurs:
```
Error: open secrets.yml: no such file or directory

  on site.tf line 30, in data "local_file" "variables":
  30: data "local_file" "variables" {
```

**Comment by mvantellingen** [comment]
Can you try `mach-composer plan --var-file=secrets.yaml` ?

**Comment by tleguijt** [comment]
Fixed with commit 8c7816f
